runtime/moonbase/src/lib.rs

```diff
@@ -1610,3 +1621,221 @@ mod tests {
 		}
 	}
 }
+
+#[cfg(test)]
+mod fee_tests {
+	use super::*;
+	use frame_support::weights::WeightToFee;
+	use frame_support::{dispatch::GetDispatchInfo, traits::OnFinalize, weights::DispatchInfo};
+	use sp_runtime::traits::Convert;
+
+	fn run_with_system_weight<F>(w: Weight, mut assertions: F)
+	where
+		F: FnMut() -> (),
+	{
+		let mut t: sp_io::TestExternalities = frame_system::GenesisConfig::default()
+			.build_storage::<Runtime>()
+			.unwrap()
+			.into();
+		t.execute_with(|| {
+			System::set_block_consumed_resources(w, 0);
+			assertions()
+		});
+	}
+
+	#[test]
+	fn test_multiplier_can_grow_from_zero() {
+		let minimum_multiplier = MinimumMultiplier::get();
+		let target = TargetBlockFullness::get()
+			* BlockWeights::get()
+				.get(DispatchClass::Normal)
+				.max_total
+				.unwrap();
+		// if the min is too small, then this will not change, and we are doomed forever.
+		// the weight is 1/100th bigger than target.
+		run_with_system_weight(target * 101 / 100, || {
+			let next = SlowAdjustingFeeUpdate::<Runtime>::convert(minimum_multiplier);
+			assert!(
+				next > minimum_multiplier,
+				"{:?} !>= {:?}",
+				next,
+				minimum_multiplier
+			);
+		})
+	}
+
+	#[test]
+	#[ignore]
```

Comment:
Why is this ignored?

Reply:
I think because it's a long-running test. As with your other comment, these came from substrate a long time ago :)

Reply:
Yeah I copied it over from substrate.